app/elements/youneeq-tracker/youneeq-tracker.js

```diff
@@ -17,7 +17,7 @@ class youneeqTracker {
       },
       previousURL: {
         type: String,
-        value: ''
+        value: document.referrer
```

Comment:
Although this works fine and Polymer will allow for this type of declaration. I am going to modify this a bit to read something more like

`previousURL: {
    type: String,
    value: function() {
        return document.referrer;
    }
}`